guarddog/analyzer/sourcecode/code-execution.yml

```diff
@@ -41,6 +41,11 @@ rules:
           - pattern: subprocess.run([..., "... $ARG1 ...", ...], ...)
           - pattern: run([..., "$ARG1", ...], ...)
 
+          - pattern: subprocess.check_call($ARG1, ...)
+          - pattern: check_call($ARG1, ...)
+          - pattern: subprocess.check_call([..., "... $ARG1 ...", ...], ...)
```

Comment:
not sure about this one, what you tried to achieve? metavar is already a regex, this should match without the "... ..." pattern.
FYI the ... thing here is very costy.
The same for the `subprocess.run` a couple lines above (that you did not add)

Reply:
It might have to do with the way the rule is written on a macro scale, but if I make these simplifications, it no longer passes the tests.